src/firefly/java/edu/caltech/ipac/firefly/server/RequestOwner.java

```diff
@@ -64,15 +64,17 @@ public RequestAgent getRequestAgent() {
 
     public void setRequestAgent(RequestAgent requestAgent) {
         this.requestAgent = requestAgent;
-        host = requestAgent.getHeader("host");
-        referrer = requestAgent.getHeader("Referer");
-
-        String sei = requestAgent.getCookie("seinfo");
-        if (!StringUtils.isEmpty(sei)) {
-            String [] parts =  sei.split("_");
-            if (parts.length > 1) {
-                eventConnID = parts[0];
-                eventChannel = parts[1];
+        if (requestAgent != null) {
```

Comment:
When RequestOwner is referenced outside of a http request thread.  Somewhere in the our cache code, it's doing it.